docs/instrument-traces/go-app.md

```diff
@@ -1,46 +1,36 @@
 # Go App
 
-### Auto-instrument sample Golang app for traces
+## Auto-instrument sample Golang app for traces
 
 In this tutorial, we will go through the steps to auto-instrument a Go app to send traces to Siglens.
 
-### Prerequisites
-- Siglens instance should be running on localhost with ingest port-4318. To do so you need to change the ingest port of Siglens to `4318` in `server.yaml`
-- Go app (refer to the documentation below if you don't have the setup for the Go app)
-
-### Set up for Go application
-
-Given below are the instructions for setting up a sample Golang application for a bookstore:
-
+## Quickstart
+Start siglens:
+```bash
+git clone https://github.com/siglens/siglens.git
+cd siglens
+go run cmd/siglens/main.go --config server.yaml
```

Comment:
we should as them to use the install script from siglens.com/trynow, it will bring up siglens and any other components we add in future (like nginx, otel, etc..)